.storybook/preview.ts

```diff
@@ -16,7 +16,8 @@ const preview: Preview = {
         method: 'alphabetical'
         // TODO: uncomment when we build out these sections
         // order: ['Getting started', 'Tokens', 'Components', 'Patterns']
```

Comment:
tokens feel a little lost at the bottom of the panel. thoughts on uncommenting this now?